nibabel/tests/test_nifti1.py

```diff
@@ -538,7 +538,7 @@ def test_slice_times(self):
         hdr.set_slice_duration(0.1)
         # We need a function to print out the Nones and floating point
         # values in a predictable way, for the tests below.
-        _stringer = lambda val: val is not None and f'{val:2.1f}' or None
+        _stringer = lambda val: (val is not None and f'{val:2.1f}') or None
         _print_me = lambda s: list(map(_stringer, s))
```

Comment:
This is just an old style ternary expression, and we can slide it into the comprehension:

```suggestion
        _print_me = lambda s: [val if val is None else f'{val:2.1f}' for val in s]
```